packages/vscode-integration-webview/src/diagram-identifier.ts

```diff
@@ -26,10 +26,21 @@ export interface GLSPDiagramIdentifier {
 
 export function isDiagramIdentifier(object: any): object is GLSPDiagramIdentifier {
     return (
-        object !== undefined &&
-        typeof object === 'object' &&
+        AnyObject.is(object) && //
         hasStringProp(object, 'clientId') &&
         hasStringProp(object, 'diagramType') &&
         hasStringProp(object, 'uri')
     );
 }
+
+// ----------------------------------
+// Initial Handshake
+
+export interface WebviewReadyMessage {
```

Comment:
Yes, exactly code duplication here is on purpose because its used in two packages that are independent from each other.
If at any point we end up with more shared code we could move it into a shared package but for now it's not worth the effort.